authors/apps/articles/views.py

```diff
@@ -355,3 +366,22 @@ class ListTagsView(ListAPIView):
     permission_classes = (IsAuthenticated,)
     renderer_classes = (TagJSONRenderer,)
     queryset = Tag.objects.all()
+
+
+class ReadStatsAPIView(ListAPIView):
+    """
+    this view class enables the author to view their reading
+    stats
+    """
+    serializer_class = ReadStatsSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, username):
+        """ this gets the readstats of an article """
+        reading_stats = ReadingStats.objects.filter(user=request.author.id)
+        reading_statsCount = reading_stats.count()
+        serializer = self.serializer_class(reading_stats, many=True)
+        return Response({
```

Comment:
Akram, use a JSON renderer instead of returning a plain message

Reply:
Worked on it